app/_partner/hp.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: partner
+lang: en
+permalink: /hp/
+
+id: hp
+name: hp
+logo:
+contact:
+
+flickr: https://www.flickr.com/
```

Comment:
I think this has to be the URL of an album and not just a link to the main flickr website.

Reply:
This button is just a place holder for them to see what they can customize, so we'll get a proper link if they choose to 1) use it and 2) provide an album.  Good catch!

Reply:
I recommend leaving the `flicker:` tag blank, but don't remove the whole line. I.e.
```
flickr:
```